MediaBrowser.Model/System/SystemStorageInfo.cs

```diff
@@ -0,0 +1,56 @@
+using System.Collections.Generic;
+
+namespace MediaBrowser.Model.System;
+
+/// <summary>
+/// Contains informations about the systems storage.
+/// </summary>
+public class SystemStorageInfo
+{
+    /// <summary>
+    /// Gets or sets the program data path.
+    /// </summary>
+    /// <value>The program data path.</value>
+    public required FolderStorageInfo ProgramDataStorage { get; set; }
```

Comment:
I think these "Storage" suffixes are redundant when the class itself already implies these properties contain storage info.

Reply:
then your previous comment on the topic was very missleading^^

Reply:
No no no read it again:

> If all the variables in here are just information about paths the "DirectoryInfo" suffix is unneeded.

I meant that a suffix would only be necessary if we also added different types of info, but we opted make this storage specific :p

Reply:
Renamed to `*Folder`